app/src/main/java/app/revanced/integrations/settingsmenu/ReVancedSettingsFragment.java

```diff
@@ -95,6 +101,36 @@ public class ReVancedSettingsFragment extends PreferenceFragment {
                     LogHelper.printException(ReVancedSettingsFragment.class, "No valid setting found: " + setting.toString());
                 }
 
+                if (setting == SettingsEnum.PREFERRED_VIDEO_QUALITY_WIFI) {
```

Comment:
Please deduplicate this code, most of the duplicate lines can be reduced to one.